src/target/cortexm.c

```diff
@@ -326,7 +356,15 @@ bool cortexm_probe(ADIv5_AP_t *ap, bool forced)
 			return false;
 
 #define PROBE(x) \
-	do { if ((x)(t)) {target_halt_resume(t, 0); return true;} else target_check_error(t); } while (0)
+	do {					\
+	  if ((x)(t)) {				\
+	    if (!conn_reset) {			\
+	      target_halt_resume(t, 0);		\
+	    }					\
+	    return true;			\
```

Comment:
Please explain in what state a target leaves here with "mon connect enable". I think it is halted and reset is released. I think reset should be reenabled.